File: examples/user/read/node/readAccountByEmail.js
import * as user from "m3o/user";

// Read an account by id, username or email. Only one need to be specified.
async function ReadAccountByEmail() {
  let userService = new user.UserService(process.env.MICRO_API_TOKEN);
  let rsp = await userService.read({
    email: "joe@example.com",
  });
  console.log(rsp);
}

await ReadAccountByEmail();

File: examples/user/read/go/readAccountByEmail.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/user"
)

// Read an account by id, username or email. Only one need to be specified.
func ReadAccountByEmail() {
	userService := user.NewUserService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := userService.Read(&user.ReadRequest{
		Email: "joe@example.com",
	})
	fmt.Println(rsp, err)
}
